docker-compose-services/postgres/README.md

```diff
@@ -38,6 +38,10 @@ Two new `ddev` commands are provided:
 Example `config.yaml` hooks configuration to automatically import/export the `db` table:
 
 ```
+# Add psql to your webserver
+webimage_extra_packages: [postgresql-client]
+
+# Automatically import and export PostreSQL database on ddev start and stop
```

Comment:
Just an FYI, the hooks to import and export aren't needed any more as it's stored in a volume in the current version.